packages/react-app/src/pages/manage.tsx

```diff
@@ -43,39 +44,22 @@ const Tabs = {
   Archived: 1,
 };
 
+const IsTabActive = (tab) => !!!tab;
+
 const Manage = () => {
   const userInfo = useContext(UserContext) as UserInfo;
-  const { isLoading, data, refetch } = useGiveaways();
   const [activeTab, setActiveTab] = useState(Tabs.Active);
+  const { isLoading, data, refetch } = useGiveaways({
+    active: IsTabActive(activeTab),
+  });
   const [countdownGiveaway, setCountdownGiveaway] = useState<Giveaway | null>();
   const [showConfettis, setShowConfettis] = useState(false);
   const [error, setError] = useState(false);
 
-  const giveaways = useMemo(() => {
-    return data?.filter((g) => {
-      const now = new Date();
-      const giveawayStartDate = new Date(g.startTime);
-      const giveawayEndDate = new Date(g.endTime);
-
-      if (userInfo.role !== UserRole.ADMIN && giveawayStartDate > new Date()) {
-        return false;
-      }
-
-      if (g._id === countdownGiveaway?._id) {
-        return false;
-      }
```

Comment:
I believe we are missing this part, this is just to avoid displaying the same giveaway twice